doc/clang.txt

```diff
@@ -265,6 +265,7 @@ A: Please use with neocomplete, they can work well together.
         if !exists('g:neocomplete#force_omni_input_patterns')
           let g:neocomplete#force_omni_input_patterns = {}
         endif
+        let g:neocomplete#force_overwrite_completefunc = 1
```

Comment:
You can create clang source for deoplete.
deoplete can complete both neosnippet and clang candidates.
But the feature is not implemented yet.